### PR TITLE
Added maven wrapper

### PR DESCRIPTION
**Pull request by NikolasKomonen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@ target/
 *~
 
 .gradle
+.vscode/
```
Comment (fbricon) [inline .gitignore]: don't ignore .vscode

but ignore .mvn/wrapper/maven-wrapper.jar

Comment (fbricon) [reply .gitignore]: the maven-wrapper.jar will be downloaded automatically if missing from the directory, so no need to pollute the git repo with that binary

File: .vscode/launch.test.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "run": {
+        "default": "",
+        "items": [
+            {
+                "name": "xml-languageserver",
+                "projectName": "xml-languageserver",
+                "workingDirectory": "/home/nkomonen/Documents/angelo-xmlLS/xml-languageserver",
```
Comment (fbricon) [inline .vscode/launch.test.json]: "workingDirectory": ""

File: .vscode/launch.test.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            }
+        ]
+    },
+    "debug": {
+        "default": "",
+        "items": [
+            {
+                "name": "xml-languageserver",
+                "projectName": "xml-languageserver",
+                "workingDirectory": "/home/nkomonen/Documents/angelo-xmlLS/xml-languageserver",
```
Comment (fbricon) [inline .vscode/launch.test.json]: "workingDirectory": ""

**Comment by fbricon** [comment]
please remove .mvn/wrapper/maven-wrapper.jar from the commit and fix the "workingDirectory" fields in launch.test.json

File: .vscode/launch.test.json (hunk 3 above)
Comment (NikolasKomonen) [reply .vscode/launch.test.json]: I cleaned everything up and this file is not here anymore. Everything works fine though.

File: .vscode/launch.test.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "run": {
+        "default": "",
+        "items": [
+            {
```
Comment (fbricon) [inline .vscode/launch.test.json]: replace all xml-languageserver with lsp4xml

**Pull request merged**